plugins/search/service/include/service/serviceprocess.h

```diff
@@ -61,6 +61,7 @@ class ServiceProcess : public SearchServiceIf, public util::PipedProcess
       ::execlp("java", "java", "-server",
         //"-Xdebug", "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=8666",
         "-Djava.util.logging.config.class=cc.search.common.config.LogConfigurator",
+        "-Djava.util.logging.SimpleFormatter.format=[%4$s] %5$s%6$s%n",
```

Comment:
I couldn't find a solution to write the log level in lowercase here, so I wrote the log level in uppercase at cpp.